### DISCUSSION TITLE
Welcome to LangGraph-GUI

### DISCUSSION THREAD

**Discussion by HomunMage** [LangGraph-GUI/LangGraph-GUI#1] [Announcements]
Welcome to LangGraph-GUI

**Comment by deodharaditi** [comment]
I have installed and run everything frontend, backend and Ollama server. I was wondering how I can start using the GUI?

**Reply by HomunMage** [reply]
Thank you for ask the issue
you can try some examples: https://langgraph-gui.github.io/Frontend/Examples
and Node explain https://langgraph-gui.github.io/Frontend/Graph
I know these are not good enough. If you intresting, please ask more that i can make the doc better

**Comment by ved-ssa** [comment]
Hi I have setup the project and able to run backend, frontend and the Ollama (with timyllama model). But now when I load the example graphs and try to run them, it just fails and shows blank error message ```data: {'status': 'error', 'message': ''} ```

Can someone give instructions on running the examples. or give more clear guides on creating and running graphs?

**Reply by HomunMage** [reply]
thanks for asking
I will record video this weekend for better starting on this repo

**Reply by ved-ssa** [reply]
Thank you for prompt reply. Just wanted to let you know that I have setup this on a Windows 11 machine. I want some steps to get started with running the workflows. Thanks again. :)

**Reply by HomunMage** [reply]
oh, I'm not familiar with Win11.  I cannot reproduce the blank error message. I don't know how to fix your issue.

That's maybe Win11 some problem, I will recommend use WSL2 with vscode wsl that vscode will auto do port forward for you.

If possible, please provide more log of python server part.
